### ISSUE TITLE
Citrus validates text content although I did not tell it

### ISSUE THREAD

**Issue by Ernesto-Maserati-CH** [christophd/citrus#197]
I have a plain-text inbound message and want only to verify the HTTP code. But that does not seem to work.
```
       http(action -> action.client(CLIENT_ID)
                .response(HttpStatus.OK)
        );
```
This results in:

> com.consol.citrus.exceptions.CitrusRuntimeException: org.w3c.dom.ls.LSException: Content is not allowed in prolog.

What is the matter?

**Comment by christophd** [comment]
XML is the default message type so Citrus tries to parse the payload with XML parser in order to perform XML validation. You need to explicitly set the message type to PLAINTEXT.

**Comment by Ernesto-Maserati-CH** [comment]
I assumed the HTTP status code is not related to the message type.
If I don't care about the message type when I want to validate only the status code, why does Citrus have to care about it?

**Comment by christophd** [comment]
You are right, message payload validation should be skipped here automatically.

**Comment by christophd** [comment]
Fixed with #218
